apidef/oas/server.go

```diff
@@ -42,7 +42,7 @@ func (s *Server) Fill(api apidef.APIDefinition) {
 		s.GatewayTags = &GatewayTags{}
 	}
 	s.GatewayTags.Fill(api)
-	if ShouldOmit(s.GatewayTags) {
+	if len(s.GatewayTags.Tags) == 0 {
```

Comment:
Can you please explain why we change it ?

Reply:
Think that the object will have more fields in future. That’s why we should continue with shouldomit.

Reply:
`shouldOmit` won't make `s.GatewayTags` when `s.GatewayTags.Enabled` is `true` and `len(s.GatewayTags.Tags)` is 0. I think this is required in such cases

Reply:
It is a valid case why would you omit if enabled `true`. If you don't want 0 length tags with enabled, you would handle it in schema.